src/common/utils/id-validation.util.ts

```
@@ -0,0 +1,29 @@
+  }
+
+  /**
+   * Validates multiple string IDs and returns their parsed values
+   * @param ids - Array of objects with id and optional fieldName
+   * @returns Array of parsed integer IDs
+   */
+  static validateAndParseIds(
+    ids: Array<{ id: string; fieldName?: string }>,
+  ): number[] {
+    return ids.map(({ id, fieldName = 'id' }) =>
+      this.validateAndParseId(id, fieldName),
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)